src/snapshotter.js

```diff
@@ -46,9 +48,13 @@ export default class SnapShotter {
       ? this._webdriver.Capabilities.chrome
       : this._webdriver.Capabilities.firefox;
 
+    const capability = mobileDeviceName
+      ? this.getMobileBrowserCapability()
+      : browserCapability();
+
```

Comment:
If you specify in your config Firefox and a mobile browser, it will not work. We need to just add a validation so if you have these two competing config values, throw and error and say mobile emulation is not supported in Firefox.

Reply:
cool will change :)